lib/note-detail.jsx

```diff
@@ -107,13 +99,17 @@ export default React.createClass( {
 	},
 
 	renderEditable( divStyle ) {
-		const note = this.props.note;
+		const { content = '' } = this.props.note.data;
```

Comment:
I did this actually to keep with the `renderMarkdown` method above, which does exactly this
(no chance of a `TypeError` here, see `if ( ! this.isValidNote( note ) ) return;` before it in `render`)